src/System Application/Test/MicrosoftGraph/src/GraphClientTest.Codeunit.al

```diff
@@ -179,4 +354,61 @@ codeunit 135140 "Graph Client Test"
         exit(StringBuilder.ToText());
     end;
 
+    local procedure GetSinglePageResponse(): Text
+    var
+        StringBuilder: TextBuilder;
+    begin
+        StringBuilder.Append('{');

```

Comment:
These can actually be resource files which will make changes much easier.
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-app-resources